packages/i18n/locales/cs/translation.json

```diff
@@ -0,0 +1,629 @@
+      "automaticallyAddTokensTooltip": "Měly by se tokeny zaslané do DAO přidat do pokladnice?",
+      "codeID": "Code ID",
+      "contractLabel": "Náhled kontraktu",
+      "cw20Token": "CW20 Token",
+      "daoDescriptionPlaceholder": "Popište své DAO...",
+      "daoName": "Název",
+      "daoNamePlaceholder": "Pojmenujte své DAO",
+      "defaultTierName": "Hlavní členové",
+      "description": "Popis",
+      "existingTokens": "Existující token",
+      "fromValidator": "Od validátora",
+      "funds": "Prostředky",
+      "governanceTokenNamePlaceholder": "Názevázev vašeho tokenu",
+      "governanceTokenSymbolPlaceholder": "Zkratka vašeho tokenu ",
+      "image": "Obrázek",
+      "imageURLTitle": "URL obrázku",
+      "imageURLTooltip": "Odkaz na váš obrázek, například: https://moonphase.is/image.svg",
+      "initialSupply": "Množství tokenů",
+      "instantiatedAddress": "Adresa",
+      "maxCharacters": "Maximálně {{max}} znaků",
+      "membersAddress": "Adresy členů",
+      "membersToAddOrUpdate": "Přidat nebo upravit členy",
+      "membersToRemove": "Vyhodit členy",
+      "message": "Zpráva",
+      "migrateDescription": "Toto <1>migrate</1> vybraný kontrakt na nové ID.",
+      "migrateMessage": "Zpráva migrace",
+      "name": "Název",
+      "onlyMembersExecuteTitle": "Pouze členové mohou dokončit návrhy",
+      "onlyMembersExecuteTooltip": "Pokud je povoleno, mohou odhlasované návrhy dokončovat pouze členové..",
+      "optional": "optimální",
+      "passingThresholdDescription": "Pokud vaše DAO nemá nastavené kvorum, jedná se o procento hlasovacích práv DAO, které musí hlasovat ano, aby byl návrh schválen. Například při 50% prahu pro přijetí musí být polovina hlasovací síly pro návrh, aby prošel. Pokud má vaše DAO nastavené kvorum, práh přijetí se počítá pouze z těch, kteří hlasovali. Například při kvoru 50 % a prahu přijetí 50 % může návrh projít, pokud pro něj hlasovalo pouze 25 % všech hlasujících. Doporučuje se stanovit většinový práh pro přijetí.",
+      "passingThresholdTitle": "Průchozí hranice",
+      "percentOfTotalSupply": "Procento z celkové nabídky tokenů",
+      "percentOfTotalSupplyTooltip": "{{weight}} z celkové zásoby tokenů bude rovnoměrně rozdělena mezi všechny členy této hodnosti. Chcete přidat členy s různou hlasovací silou? Přidejte další hodnosti.",
+      "proposalDepositDescription": "Počet tokenů, které je třeba vložit k vytvoření návrhu. Jeho nastavení může zabránit spamu, ale příliš vysoké nastavení může omezit širokou účast.",
+      "proposalDepositTitle": "Zástava návrhu",
+      "proposalsDescriptionPlaceholder": "Dejte svému návrhu popis...",
+      "proposalsName": "Název návrhu",
+      "proposalsNamePlaceholder": "Dejte svému návrhu název...",
+      "quorumDescription": "Minimální procento hlasů, které musí být odhlasováno, aby byl návrh projednán, pokud máte DAO s mnoha neaktivními členy, může nastavení této hodnoty příliš vysoké ztížit schvalování návrhů.",
```

Comment:
Minimální procento hlasů, které musí aktivně hlasovat, aby byl návrh projednán. Pokud máte DAO s mnoha neaktivními členy, může nastavení této hodnoty na příliš vysokou hodnotu ztížit pravděpodobnost schvalování návrhů.